README.md

```diff
@@ -57,11 +60,15 @@ If you don't use it, there is no runtime cost (except when executing a snippet o
 
 ### What's with escaping, what kind of characters can I use and not use?
 
-You shouldn't use unbalanced braces (`{}`) in snippet names, and auto-expansion triggers should be limited to alphanumeric characters. You can use `^` if you want a trigger to only match at the beginning of a line.
+You should be able to use anything. Triggers are currently restricted to at most 6 characters (at least for auto-expansion), but if you have a reasonable use case for more we can raise it. The number is arbitrary.
+
+### My snippets are expanding too greedily. If I type 'before', I don't want my 'for' snippet to be expanded.
+
+You should use a stricter trigger for the snippet. For example, `\bfor` will only expand if `for` starts at a word boundary. Similarly, you can use `^` to match the start of a line.
 
 ### Can you add snippets for language `X`?
 
-No, but you're welcome to submit your own and I'll add them. TODO
+No, but you're welcome to start collecting them in the wiki.
```

Comment:
I'm curious if this should be done this way. If #6 going to happen then it will be tedious to copy snippets from wiki to separate files. Shouldn't we have a separate repository with snippet collections for various languages and simply accept PRs with new snippets? If plugin will gain popularity (i'm sure it will), something like https://github.com/honza/vim-snippets will be really useful to have.

Reply:
I forgot to reply to this: I don't really mind how it's done, the wiki is a simple git repo afaik so it could work. The advantage is that it's easier to keep in sync with the main repo.